src/util/Constants.js

```diff
@@ -58,13 +58,15 @@ exports.DefaultOptions = {
   /**
    * WebSocket options (these are left as snake_case to match the API)
    * @typedef {Object} WebsocketOptions
-   * @property {number} [large_threshold=250] Number of members in a guild to be considered large
+   * @property {number} [large_threshold=50] Number of members in a guild to be considered large
    * @property {boolean} [compress=false] Whether to compress data sent on the connection
+   * @property {boolean} [guild_subscriptions=true] enables dispatching of guild subscription events (presence and typing events)
```

Comment:
This documentation string is incomplete.

nit 1. Capitalize Enables

As you can see at https://github.com/discordapp/discord-api-docs/pull/1016#issuecomment-510492900, turning this off right now disables... GUILD_MEMBER_*.. That could potentially break expected library behavior, or worst case scenario, the library as a whole due to the cache not being kept up to date. Also, This is probably not how we want to expose such a WS setting. It'd probably become a client setting that is linked to the ws one, with proper documentation, disclaimers, TESTING, etc.